client/components/plans/plans-compare/index.jsx

```diff
@@ -96,6 +100,53 @@ var PlansCompare = React.createClass( {
 		}, this );
 	},
 
+	showFreeTrialException: function() {
+		const hasTrial = this.props.sites && this.props.sites.hasLoadedFromServer
+				? this.props.sites.getSelectedSite().plan.free_trial
+				: false,
+			canStartTrial = this.props.sitePlans && this.props.sitePlans.hasLoadedFromServer
+				? this.props.sitePlans.data.some( property( 'canStartTrial' ) )
+				: false;
+
+		if ( ! config.isEnabled( 'upgrades/free-trials' ) ) {
+			return false;
+		}
+
+		// always show if the user is currently in trial
+		if ( hasTrial ) {
+			return true;
+		}
+
+		// show if the site is eligible for a trial (it never had a free trial before)
+		// and free trial is enabled for this component
+		if ( canStartTrial && this.props.enableFreeTrials ) {
+			return true;
+		}
+
+		// show if we are in signup and free trial is enabled for this component
+		if ( this.props.isInSignup && this.props.enableFreeTrials ) {
+			return true;
+		}
+
+		return false;
+	},
+
+	freeTrialExceptionMarker: function( feature ) {
+		if ( this.showFreeTrialException() && featuresListUtils.featureNotPartOfTrial( feature ) ) {
+			return '*';
+		}
+
+		return null;
+	},
+
+	freeTrialExceptionMessage: function( featuresList ) {
+		if ( this.showFreeTrialException() && featuresList.some( featuresListUtils.featureNotPartOfTrial ) ) {
+			return <div className="plans-compare__free-trial-exception-message">{ this.translate( '* Not included during the free trial period' ) }</div>;
```

Comment:
This line if already quite long. I think we could easily address that with indentation:

```
return (
  <div className="plans-compare__free-trial-exception-message">
    { this.translate( '* Not included during the free trial period' ) }
  </div>
);
```